assets/r/model_build_rpart.R

```diff
@@ -84,11 +84,22 @@ rattle::asRules(model_rpart)
 # Extract and print rules.
 
 rules <- rattle::asRules(model_rpart)
+
+model_rpart_tu <- rpart(
+  form,
```

Comment:
No, we don't build the model on tu. We build the model on tr and then test the performance of the model on tu.